delivery/media/cat/README.MD

```diff
@@ -0,0 +1,26 @@
+# CAT Module
+
+The CAT module can be used to perform operations related to CAT tokens such as CAT token generation and validation. The module considers CAT tokens defined as per this [spec](https://docs.google.com/document/d/1lqsu9v8RSWDtd4p8vAgkUwE5IWBxb4YX/edit). 
+It exports implementations of CAT class that contains API's to create, decode and validate CWT tokens.
+
+## Important Note
+- CAT module is reference implementation which matches with the spec as of 13th May 2024.
+- Implementation covers high level claims and customers are free to extend it to other claims in the spec as per their requirement.
+- As of now the enc claim is supported using A256-GCM encryption algorithm only and users are free to adopt to new algorithms when it is available in EW crypto
+
+## Limitations
+- The CAT module relies on the alg header claim defined in protected header of the CAT token. However the CAT spec does not mandate this, but this module uses the value defined in the protected header to identiy the algorithm used for token verification. The protected header also provides additional security. 
+- The module depends on CWT module (1.2.0 version) and its limitations are also extended. Refer [cwt](https://techdocs.akamai.com/edgeworkers/docs/cwt) doc to know its limitations.
+
+## Subfolder organization
+* **/apis**: CAT API documentation.
```

Comment:
It's called `api` (without an `s`) in the directory tree.

Reply:
Updated. Thanks!